adhocracy4/reports/templates/a4reports/emails/report_moderators.en.email

```diff
@@ -7,6 +7,7 @@
 
 {% block content %}
 {{ report.creator.username }} requested moderation of a {{ report.content_object | verbose_name }} in your project {{ report.project }} on {{ site.name }}. The reason given is "{{ report.description }}".
+<a href={{ site.url }}{{ report.content_object.get_absolute_url }}>Click here</a> to see the {{ report.content_object | verbose_name }}.
```

Comment:
The link is already in the cta_url block down below! That should be sufficient (and also work for all report-cases?!).

Reply:
Right, didn't see that. Will update